frontend/lib/pages/loc-confirmation.tsx

```diff
@@ -20,14 +20,18 @@ const LoadableConfetti = Loadable({
   webpack: () => [],
 });
 
-function LetterStatus(props: { letterRequest: AllSessionInfo_letterRequest, locPdfURL: string }): JSX.Element {
+function LetterStatus(props: {
+  letterRequest: AllSessionInfo_letterRequest,
+  locPdfURL: string,
+  title: string
+}): JSX.Element {
```

Comment:
Honestly there is probably something to be said about CC's complaint that this function is a bit long and could use refactoring, but omg I'm tired let's :shipit: !